src/index.ts

```diff
@@ -326,6 +332,28 @@ export async function install(
     };
     const packageBundle = await rollup.rollup(inputOptions);
     await packageBundle.write(outputOptions);
+
+    if (include && nomodule) {
+      // Resolve `/web_modules/` relative to package.json rather than system root
+      function renameImportPlugin() {
+        return {
+          name: 'rename-import-plugin',
+          resolveId(source) {
+            if (source.startsWith(path.sep)) {
+              return {id: source.replace(path.sep, path.join(cwd, path.sep))};
```

Comment:
I was still having issues where Rollup thought `/web_modules/` was at the system root. We’re still replacing imports here, but this time we’re trying to resolve them relative to `package.json`

Reply:
I also realized in doing the "bare module specifier" scanner work that some people may be using import maps, which we should also handle. So I guess our logic should be something like:

- if `package-name`, read from the import map and rewrite. (we can run this nomodule step last, so that we'll already have the import map data object in memory)
- if `.*/web_modules/(.*)`, rewrite to `path.join(cwd, 'web_modules', $2)` (where `$2` is everything after web_modules)
- if `/.*` and doesn't include web_modules... warn? rewrite to cwd? ignore it?
- otherwise, ignore it.

Reply:
> if `package-name`, read from the import map and rewrite. (we can run this nomodule step last, so that we'll already have the import map data object in memory)

Cool. I think we can pass that to Rollup just fine.

> if `.*/web_modules/(.*)`, rewrite to `path.join(cwd, 'web_modules', $2)` (where `$2` is everything after web_modules)

I think this wouldn’t work with `--dest` (e.g. `--dest dist/web_modules`). Since we have access to that, maybe we should try `path.join(cwd, dest)`? But keep the logic where it only applies to `web_modules`.

> if `/.*` and doesn't include web_modules... warn? rewrite to cwd? ignore it?

🤔 Since Rollup will try and resolve everything, I think what I’ll do is wrap the whole thing in a try/catch and give a friendlier error if Rollup resolution fails. The difficult thing here is the resolution is different than the include script. In the include script, if we can’t resolve it, we can’t resolve it. We move on. But here, Rollup will fail if it can’t resolve it. So all that to say it’s hard to say that everyone’s code is Rollup-able without… a Rollup config. Maybe we can get some grace here because this is an optional flag?

Reply:
+1, lets see how well this works. I would hope all JS imports are either relative or `/web_modules/*`, am I missing something obvious that could kill it or are you just worried about weird edge cases? 

If anyone needs a custom Rollup config, they probably already need a full app bundler, in which case they can manage their own legacy bundle.

Reply:
Weird edge cases, basically. And you do raise a good point! If they have legacy code, then they likely can’t use snowpack to begin with. It is nice dealing with a more predictable subset of JS code.